src/plots/gl3d/scene.js

```diff
@@ -714,6 +714,7 @@ proto.setConvert = function() {
     for(var i = 0; i < 3; ++i) {
         var ax = this.fullSceneLayout[axisProperties[i]];
         Axes.setConvert(ax, this.fullLayout);
+        ax.setScale = Lib.noop;
```

Comment:
We need it here to make margin `relayout` calls work. 

I got carried away in #1431 where I :hocho: ed [it](https://github.com/plotly/plotly.js/pull/1431/files#diff-dd23546f2ee4a55150cd584d026e1c93L18). My mistake.